packages/nocodb/src/lib/db/sql-client/lib/pg/pg.queries.ts

```diff
@@ -150,11 +150,6 @@ AND t.table_name=?;`,
 
   relationList: {
```

Comment:
One think I'm not sure about, what is the difference between pg queries here and in the PgClient.ts file?
Is this legacy code?

Reply:
We are no longer using it, so you can ignore it.